playbooks/openshift-master/private/upgrade.yml

```diff
@@ -3,6 +3,22 @@
 # Upgrade Masters
 ###############################################################################
 
+# Some change makes critical outage on current cluster.
+- name: Confirm upgrade will not make critical changes
+  hosts: oo_first_master
+  tasks:
+  - name: Confirm Reconcile Security Context Constraints will not change current SCCs
+    command: >
+      {{ openshift_client_binary }} adm policy --config={{ openshift.common.config_base }}/master/admin.kubeconfig reconcile-sccs --additive-only=true -o name
```

Comment:
Does this occur before the control plane has been upgraded or after?  If a "new" `oc` binary is used, it could false positive on this check because *we* changed the meaning of the SCCs, not the customer.  So this check must be performed with an "old" `oc` binary before any upgrade steps are taken.  This guarantees that the diff between the SCCs in caused by customer action.

Reply:
This occurs before new oc binaries are placed on the host, yes.